packages/react-core/src/components/Timestamp/__tests__/__snapshots__/Timestamp.test.tsx.snap

```diff
@@ -4,7 +4,6 @@ exports[`Matches snapshot 1`] = `
 <DocumentFragment>
   <span
     class="pf-c-timestamp"
-    datetime="2022-01-01T00:00:00.000Z"
```

Comment:
Shouldn't there still be a datetime by default in the snapshot?

Reply:
Ah yeah there still is, it's only on the `time` element now, though.

Reply:
ah - it was part of the hidden lines by github